plugins/woocommerce/templates/block-notices/notice.php

```diff
@@ -26,7 +26,7 @@
 ?>
 
 <?php foreach ( $notices as $notice ) : ?>
-	<div class="wc-block-components-notice-banner is-info"<?php echo wc_get_notice_data_attr( $notice ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?> role="alert">
+	<div class="wc-block-components-notice-banner is-info"<?php echo wc_get_notice_data_attr( $notice ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
```

Comment:
Trying to make sense of the mozilla doc here.. why not using `role="status"` instead? That won't scroll into view, but would still make sure this would be picked up properly by screen readers right?

Reply:
Either way there's a template version bump required 🫡

Reply:
Good call https://github.com/woocommerce/woocommerce/pull/51651/commits/9d29c91d9e5990f8aa916179831da1d43e3990ce